docs/metadata.rst

```diff
@@ -153,7 +153,7 @@ explanatory purposes.
             "constantOptimizer": false,
             "cse": false,
             "deduplicate": false,
-            // inliner defaults to "true"
+            // inliner defaults to "false"
             "inliner": true,
```

Comment:
These values are supposed to show the defaults.
```suggestion
            "inliner": false,
```

Reply:
Figured this wasn't the case as `jupdestRemover` should also have been `false`, but I see someone already opened a PR for this.

Reply:
No, why? `jupdestRemover` is enabled by default so it should stay at `true`.

Reply:
[Is it?](https://github.com/ethereum/solidity/blob/develop/libsolidity/interface/OptimiserSettings.h#L132) Or is this another thing completely?

Reply:
It's the same thing, but defaults are initialized here and it's initialized to `true`: https://github.com/ethereum/solidity/blob/develop/libsolidity/interface/OptimiserSettings.h#L69-L76

But I see where the confusion is coming from. I myself pointed at the struct definition when I was showing that inliner is not enabled. I should have pointed at this function instead.

Reply:
Also, to clarify further `OptimiserSettings::minimal()` corresponds to no `--optimize` flag and `OptimiserSettings::standard()` is the `--optimize` flag.

We technically also have `OptimiserSettings::none()` and `OptimiserSettings::full()` but we don't really use them outside of tests. `none()` is the only one that would not enable jumpdest remover and peephole optimizer.

Reply:
Ugh, OK, that is a bit confusing. Guess I'll close https://github.com/ethereum/solidity/pull/14273.

Reply:
WTF, is that PR even coming from a human? :) The title is just a copy of this PR's title and the only commit seems to be an empty merge :)

Reply:
> These values are supposed to show the defaults.

Fixed it.